pom.xml

```diff
@@ -388,7 +387,7 @@
                 <executions>
                     <execution>
                         <goals>
-                            <goal>validate</goal>
+                            <goal>format</goal>
```

Comment:
I'm totally anti this change.

CI should not be formatting code.  unless CI start to push the formatted code back, which doesn't seem expected either